### PR TITLE
Remove withAb option

### PR DESCRIPTION
**Pull request by jamesnicholls**
As of v8.7.1, n-flags-client no longer takes a `withAb` option. See also https://github.com/Financial-Times/n-flags-client/pull/157.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,14126 @@
 {
   "name": "@financial-times/n-express",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (apaleslimghost) [inline package-lock.json:4]: **issue**: this lockfile was generated with npm 7, which we don't fully support yet. could you revert this change please?

File: main.js [hunk 2]
```diff
@@ -51,7 +51,6 @@ const getAppContainer = (options) => {
 		{
 			withBackendAuthentication: true,
 			withFlags: false,
-			withAb: false,
```
Comment (apaleslimghost) [inline main.js:54]: **suggestion (non-blocking)**: could we add a deprecation notice so that apps still passing in this option get prompted to remove it?

**Review by apaleslimghost — CHANGES_REQUESTED** [review]
thanks @jamesnicholls! one issue to address, and one non-blocking suggestion

File: main.js (hunk 2 above)
Comment (jamesnicholls) [reply main.js:54]: Yep, good idea, will add.

File: package-lock.json (hunk 1 above)
Comment (jamesnicholls) [reply package-lock.json:4]: Whoops, will remove.

**Review by apaleslimghost — APPROVED** [review]
fantastic, thank you!
